napari_allencell_annotator/util/image_utils.py

```diff
@@ -34,8 +35,8 @@ def __init__(self, filepath: Path):
         else:
             self._image = BioImage(filepath, reader=bioio_imageio.Reader)
 
-    def get_image_data(self) -> np.ndarray:
+    def get_image_data(self) -> da.Array:
         """
-        Returns image data
+        Returns image data as a dask array
         """
-        return self._image.data
+        return self._image.get_dask_stack()
```

Comment:
can you write a test for this and change the call to 
`self.viewer.add_image(ImageUtils(path).get_image_data())

Reply:
call changed in https://github.com/aics-int/napari-allencell-annotator/pull/77/commits/e15daf5454886ffea3c70da81f200a8ee8c44a4a